src/main/java/frc/robot/commands/ShootOut.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ShooterSystem;
+
+public class ShootOut extends Command {
+    private final ShooterSystem shooterSystem;
+
+    public ShootOut(ShooterSystem shooterSystem){
+        this.shooterSystem = shooterSystem;
+
+        addRequirements(shooterSystem);
+    }
+    public void initialize() {}
+
+
+    public void execute() {
+
+    shooterSystem.rotate();
+
+    }
```

Comment:
Please press CTRL+ALT+L to reformat your code

Reply:
Good point. Please do this.

Reply:
done